app/index.html

```diff
@@ -1,24 +1,35 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>My Site | Real Dev Squad</title>
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     {{content-for "head"}}
 
-    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/website-my.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css" />
+    <link
+      integrity=""
+      rel="stylesheet"
+      href="{{rootURL}}assets/website-my.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     {{content-for "head-footer"}}
   </head>
   <body>
     {{content-for "body"}}
 
     <script src="{{rootURL}}assets/vendor.js"></script>
     <script src="{{rootURL}}assets/website-my.js"></script>
+    <script
+      src="https://kit.fontawesome.com/6b11c07424.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Umm.. Why this one? Please check the line 18,19,20,21
We need to get rid of this CDN as well, we already have a library, but nothing urgent right now

Reply:
The font awesome icons require this

Reply:
This shouldn't be required. We already have the library:
[We're already having a library for fontawesome, ](https://github.com/Real-Dev-Squad/website-my/blob/3886d862c887272e22ba7e3e65a08bb38b9b5ce8/package.json#L33)

Reply:
Is this the library which we are using ??
https://github.com/FortAwesome/ember-fontawesome

Reply:
I am using the icons directly from font-awesome.
https://fontawesome.com/v5.15/icons/bars?style=solid

Reply:
That's fine, leave the library for now, considering the time constraints.
But why are we importing fontawesome script twice?
![image](https://user-images.githubusercontent.com/56217868/148089185-97f2ad1a-2ab0-4d27-b775-9a42e566f9f1.png)

Reply:
Ok we can remove the first one in head tag, it's not needed